judgels-backends/jophiel/jophiel-app/src/main/java/judgels/jophiel/user/UserResource.java

```diff
@@ -84,6 +86,15 @@ public User createUser(AuthHeader authHeader, UserData data) {
         return userStore.createUsers(data);
     }
 
+    @Override
+    @UnitOfWork
+    public List<User> exportUsers(AuthHeader authHeader, List<String> usernames) {
```

Comment:
Can we have integration test for this?

Maybe we can write a `UserServiceDumpIntegrationTests` to be consistent with the contest's.